src/frontend/src/components/AlertAccordion/index.tsx

```diff
@@ -16,10 +18,24 @@ export default function AlertAccordion({
   severity,
   className,
 }: Props): JSX.Element {
+  const [isShown, setIsShown] = useState(false);
+
+  function handleClick() {
+    setIsShown((prevState) => !prevState);
+  }
+
   return (
-    <Alert className={className} severity={severity} icon={<ErrorOutline />}>
-      <div>{title}</div>
-      {message}
+    <Alert
+      className={className}
+      severity={severity}
+      action={
+        <IconButton aria-label="expand" color="inherit" onClick={handleClick}>
+          <ExpandMore fontSize="inherit" />
+        </IconButton>
+      }
+    >
+      {title && <AlertTitle>{title}</AlertTitle>}
+      {isShown && message}
```

Comment:
would message be an array of strings if we want to trigger the accordion?

Reply:
sorry again meant error states table

Reply:
I can add that in too if this PR was for the accordion only! I was getting ahead of myself i guess lol